app/src/main/java/io/github/jbellis/brokk/gui/dialogs/ImportDependencyDialog.java

```diff
@@ -591,4 +581,16 @@ public FileVisitResult visitFile(Path file, BasicFileAttributes attrs) throws IO
         });
     }
 
+    private static void deleteRecursively(Path path) throws IOException {
+        if (!Files.exists(path)) return;
+        try (Stream<Path> walk = Files.walk(path)) {
+            walk.sorted(Comparator.reverseOrder()).forEach(p -> {
+                try {
+                    Files.delete(p);
+                } catch (IOException e) {
+                    logger.warn("Failed to delete path during recursive cleanup: {}", p, e);
+                }
+            });
+        }
+    }
```

Comment:
We already have a few of these throughout the code, maybe extract it into some util file so we can reuse the single one in the future?

Reply:
I tried it and seems to be working fine. A couple of improvements:
- Repo URL input field is too small, I would place Load Tags & Branches button below it.
- Default branch (like `main` or `master`) should be selected by default.